Assets/MRTK/Core/Definitions/Devices/MixedRealityControllerVisualizationProfile.cs

```diff
@@ -154,24 +154,33 @@ public GameObject GlobalRightHandVisualizer
         /// </summary>
         public MixedRealityControllerVisualizationSetting[] ControllerVisualizationSettings => controllerVisualizationSettings;
 
-        /// <summary>
-        /// Gets the override model for a specific controller type and hand
-        /// </summary>
-        /// <param name="controllerType">The type of controller to query for</param>
-        /// <param name="hand">The specific hand assigned to the controller</param>
-        public GameObject GetControllerModelOverride(Type controllerType, Handedness hand)
+        private MixedRealityControllerVisualizationSetting? GetControllerVisualizationDefinition(Type controllerType, Handedness hand)
         {
             for (int i = 0; i < controllerVisualizationSettings.Length; i++)
             {
                 if (SettingContainsParameters(controllerVisualizationSettings[i], controllerType, hand))
                 {
-                    return controllerVisualizationSettings[i].OverrideControllerModel;
+                    return controllerVisualizationSettings[i];
                 }
             }
-
             return null;
         }
 
+        public bool ControllerVisualizationDefinitionExists(Type controllerType, Handedness hand)
```

Comment:
Is this used? If so, would be good to add a `<summary>` tag to clarify when it should be used. If not, can it be removed?